keyboards/nomu30/rules.mk

```diff
@@ -31,3 +31,5 @@ BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
+
```

Comment:
Lines 1-34 of this file should be deleted

Reply:
Deleted 336a70c04103ff7a407e1ec096cdf6a6b4c2abb9